msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -29,6 +29,40 @@ The remainder of this article walks you through the structure and latest updates
 
 TeamsJS v.2.0 introduces the ability for certain types of Teams apps to run across the Microsoft 365 ecosystem. The other Microsoft 365 application hosts (including Microsoft 365 app and Outlook) for Teams apps support a subset of the application types and capabilities you can build for the Teams platform. This support expands over time. For a summary of host support for Teams apps, see [TeamsJS capability support across Microsoft 365](../../m365-apps/teamsjs-support-m365.md).
 
+### Improve app performance with tree shaking
```

Comment:
"app performance" kind of sounds like runtime perf. How about:

Improve load time performance with JavaScript tree shaking

Reply:
Updated in next commit.